modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc

```diff
@@ -130,31 +130,30 @@ spec:
 .Example: YAML for a task in the pipeline
 [source,yaml]
 ----
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: mvn
 spec:
   workspaces:
   - name: maven-repo
-  inputs:
-    params:
-    - name: GOALS
-      description: The Maven goals to run
-      type: array
-      default: ["package"]
-    resources:
+  resources:
+    inputs:
     - name: source
       type: git
+  params:
+  - name: GOALS
+    description: The Maven goals to run
+    type: array
+    default: ["package"]
   steps:
     - name: mvn
       image: gcr.io/cloud-builders/mvn
       workingDir: /workspace/source
       command: ["/usr/bin/mvn"]
       args:
         - -Dmaven.repo.local=$(workspaces.maven-repo.path)
-        - "$(inputs.params.GOALS)"
-      priorityClassName: pipeline1-pc
```

Comment:
why was this line removed? this causes the priority class to _not_ get applied to this task

Reply:
ah, i understand 😞
we need to set it in the pipelinerun then, and also set the limitrange 🤔
@vdemeester @ppitonak let's add it to the pipelinerun and also mention that users need to either set limitrange or specify resources explicitly, wdyt?
users need to play with both, limitrange and priority class to make this work.

Reply:
yes.